feature/experimental/policy/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,311 @@
+# RT-3.33: DSCP based traffic steering from Non-default VRF to Default VRF using Policy based VRF selection
+
+
+## Summary
+In this case, traffic is gradually moved from the Default VRF to the non-Default VRF per traffic class and then move traffic from Non default VRF to Default VRF based on DSCP value & destination prefix.
+
+## Testbed type
+  [atedut_4.binding](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.binding)
+
+## Procedure
+Test environment setup<br><br>
+
+### Topology
+Create the following connections:
+```mermaid
+graph LR; 
+subgraph DUT [DUT]
+    B1[Port1]
+    B2[Port2]
+    B3[Port3]
+end
+subgraph ATE2 [ATE2]
+    C2[Port2]
+    C3[Port3]
+end
+A1[ATE1:Port1] <-- IBGP(ASN100) --> B1; 
+B2 <-- EBGP(ASN100:ASN200) --> C2;
+B3 <-- EBGP(ASN100:ASN300) --> C3;
+```
+### Advertisements:
+   * ATE1:Port1 advertises following prefixes over IBGP to DUT:Port1.
+     * IPv4Prefix1/24 IPv6Prefix1/64
+     * IPv4Prefix2/24 IPv6Prefix2/64
+     * IPv4Prefix3/24 IPv6Prefix3/64
+     * IPv4Prefix4/24 IPv6Prefix4/64
+     * IPv4Prefix5/24 IPv6Prefix5/64<br>
+
+   * ATE2:Port2 and DUT:Port2 from EBGP neighborship between each other over on respective AFI-SAFI connections and advertise the following prefixes
+     * From ATE2:Port2 to DUT:Port2
+       * IPv4Prefix6/24 IPv6Prefix6/64
+       * IPv4Prefix7/24 IPv6Prefix7/64
+       * IPv4Prefix8/24 IPv6Prefix8/64
+       * IPv4Prefix9/24 IPv6Prefix9/64
+       * IPv4Prefix10/24 IPv6Prefix10/64
+     * From DUT:Port2 to ATE2:Port2
+       * All the IBGP learnt prefixes from Prefix1 to Prefix5<br>
+
+   * ATE2:Port3 and DUT:Port3 from EBGP neighborship between each other over on respective AFI-SAFI connections and advertise the following prefixes
+     * From ATE2:Port2 to DUT:Port2
+       * IPv4Prefix6/24 IPv6Prefix6/64
+       * IPv4Prefix7/24 IPv6Prefix7/64
+       * IPv4Prefix8/24 IPv6Prefix8/64
+       * IPv4Prefix9/24 IPv6Prefix9/64
+       * IPv4Prefix10/24 IPv6Prefix10/64
+     * From DUT:Port2 to ATE2:Port2
+       * All the IBGP learnt prefixes from Prefix1 to Prefix5<br>
+
+
+     * **[Important: Please note]** Following advertisements must happen only during the test one at a time as instructed<br>
+
+* Configure the DUT to honor the TOS byte of destination 
```

Comment:
How is this specified? Is there are need for an egress classification/DSCP rewrite policy to be applied to the DUT?